BUILD

```diff
@@ -1301,3 +1301,14 @@ exports_files([
     "conformance/text_format_failure_list_java.txt",
     "conformance/text_format_failure_list_java_lite.txt",
 ])
+
+filegroup(
```

Comment:
We should stick to _either_ `filegroup` or `exports_files`. (Regardless of current behavior, we should not add ambiguity in mapping between files and rules.)

Reply:
I feel this is a common pattern even in Bazel itself but maybe I don't think I understand the ambiguity you're referring to.

Either way, the filegroup is now deleted.

Reply:
Sorry, after doing more digging it seems the filegroup is what's used in [Bazel itself](https://github.com/bazelbuild/bazel/blob/7ce19e4128f7a5db219623aadf448d5690097140/tools/build_defs/BUILD#L29-L39). I've removed the exports and restored the filegroup.